apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -88,6 +88,12 @@ internal Settings(Settings other)
             /// </summary>
             public FlowControlSettings FlowControlSettings { get; set; }
 
+            /// <summary>
+            /// If set to true, disables enforcing flow control settings at the Cloud PubSub server
+            /// and uses the less accurate method of only enforcing flow control at the client side.
+            /// </summary>
+            public bool UseLegacyFlowControl { get; set; } = false;
```

Comment:
If this property effectively makes the FlowControlSettings property redundant/ignored, it would probably be worth stating that in this property *and* in the FlowControlSettings property above. It's possible I've misunderstood though.